### DISCUSSION TITLE
How to "alter" the output

### DISCUSSION THREAD

**Discussion by MalfuncEddie** [ansibleguy76/ansibleforms#127] [Q&A] [answered]
Hi,

I've got a number field where one can select the size of a disk.

unfortunately for me the number needs to be appended by "G"

so field

10

should be 
extravars:
{
"disk": "10G"
}

I am probably missing this in the documentation but is this possible?

**Comment by vcmirko** [comment]
where is the number coming from ?  rest api ?
in that case use jq to append the G.

fn.fnRestBasic('get','uri','','[.records | {name:.name, disk:(.disk + "G")}]')

You can also test this on jqplay.org

**Reply by vcmirko** [reply]
https://jqplay.org/s/wxfA5cgY472

**Reply by MalfuncEddie** [reply, marked as answer]
nevermind I found it

  - type: number
    name: diskhide
    label: Disk
    noOutput : true
  - name: disk
    type: expression
    expression: "$(diskhide) + 'G'"
    runLocal: true  
    hide: true
